### DISCUSSION TITLE
Is there any way I can remove the <script id="gatsby-chunk-mapping"></script>?

### DISCUSSION THREAD

**Discussion by cristianCeamatuAssist** [gatsbyjs/gatsby#33036] [Help]
Hello! I need to remove the `<script id="gatsby-chunk-mapping"></script>` script because it is affecting the CRP (SEO critical render path) on a website with hundreds of pages, it is an inline script that needs to be parsed before rendering the rest of the page so .
Is there any way I can do that?

**Comment by juanbiberretta** [comment]
I think you can use `onInitialClientRender` on `gatsby-browser.js` to remove it like this
```js
 var chunk = window.document.getElementById("gatsby-chunk-mapping");

  if (chunk) {
    chunk.parentNode.removeChild(chunk);
  }
```

On server side you might be able to use a function from `gatsby-ssr` ?
